tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/record/RecordDefNegativeTest.java

```diff
@@ -0,0 +1,39 @@
+/*
+ *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package org.ballerinalang.test.record;
+
+import org.ballerinalang.test.util.BAssertUtil;
+import org.ballerinalang.test.util.BCompileUtil;
+import org.ballerinalang.test.util.CompileResult;
+import org.testng.annotations.Test;
+
+/**
+ * Test cases for user record definition negative test.
+ */
+public class RecordDefNegativeTest {
```

Comment:
Do we need a separate class for this? Can't we just add these cases to https://github.com/ballerina-platform/ballerina-lang/blob/v1.1.0/tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/record/OpenRecordNegativeTest.java#L81

Reply:
This test case doesn't just belong to OpenRecord (or CloseRecords). Hence I thought it'd make sense this way.

Reply:
But we seem to be having tests for the same thing in two different places, which is not right IMO. Can we at least move those tests here?

Reply:
+1 will move those here

Reply:
I have removed the mentioned duplicate-key tests, as it is tested from the tests I have added.